core/tests/webdriverio_utils/DiagnosticTestPage.js

```diff
@@ -0,0 +1,151 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the diagnostic test page, for use
+ * in Webdriverio tests.
+ */
+
+var action = require('./action.js');
+var waitFor = require('./waitFor.js');
+
+var DiagnosticTestPage = function() {
+  var startDiagnosticTestButton = $('.e2e-test-start-diagnostic-test');
+  var addNewClassroomButton = $('.e2e-test-add-new-classroom-config');
+  var newClassrooomNameInput = $('.e2e-test-new-classroom-name');
+  var newClassroomUrlFragmentInput = $('.e2e-test-new-classroom-url-fragment');
+  var createNewClassroomButton = $('.e2e-test-create-new-classroom');
+  var recommendedTopicSummaryTile = $(
+    '.e2e-test-recommended-topic-summary-tile');
+  var recommendedTopicSummaryTilesSelector = function() {
+    return $$('.e2e-test-recommended-topic-summary-tile');
+  };
+  var classroomTileSelector = $('.e2e-test-classroom-tile');
+  var editClassroomConfigButton = $('.e2e-test-edit-classroom-config-button');
+  var addTopicToClassroomButton = $('.e2e-test-add-topic-to-classroom-button');
+  var addTopicToClassroomInput = $('.e2e-test-add-topic-to-classroom-input');
+  var submitTopicIdToClassroomButton = $(
+    '.e2e-test-submit-topic-id-to-classroom-button');
+  var saveClassroomConfigButton = $('.e2e-test-save-classroom-config-button');
+  var classroomTileContainerSelector = $('.e2e-test-classroom-tile-container');
+
+  this.startDiagnosticTest = async function() {
+    await waitFor.pageToFullyLoad();
+    await waitFor.elementToBeClickable(
+      startDiagnosticTestButton,
+      'Start diagnostic test button taking too long to be clickable'
+    );
```

Comment:
ditto: unnecessary. I'm going to stop commenting on all these cases, but please resolve the others like this too

Reply:
Removed